File: Unidad 3/C#/Programa_12.cs
using System;
namespace Programa_12;

class Par_impar{
    static void Main(string[] args){
        int c, num;

        Console.Write("Cantidad de numeros a evaluar: ");
        c = Convert.ToInt32(Console.ReadLine());
        
         for(int i=1;<=c;i++){
            Console.Write("Digite un numero: ");
            num = Convert.ToInt32(Console.ReadLine());
            if(num%2==0){
                Console.WriteLine(num+" es par");
            }else{
                Console.Write(num+" es impar");
            }
         }
    }
}

File: Unidad 3/Java/Programa_12.java

package com.mycompany.programa_12;
import java.util.Scanner;
/**
 *
 * @author LaboratorioLTI2
 */
public class Programa_12 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int c, num;
        System.out.print("Cantidad de números a verfificar: ");
        c = sc.nextInt();
        
        for(int i=1;i<=c;i++){
            System.out.print("Digite un numero: ");
            num=sc.nextInt();
            if(num%2==0){
                System.out.println(num+" es par");
            }else{
                System.out.println(num+" es impar");
            }
        }
        
    }
}
